local-libs/vcluster/vclusterops/create_db.go

```diff
@@ -755,7 +755,7 @@ func (vcc VClusterCommands) produceAdditionalTLSInstructions(options *VCreateDat
 		nmaSetHTTPSTLSOp, err := makeNMASetTLSOp(&options.DatabaseOptions, httpsTLSKeyPrefix,
 			true, // grantAuth
 			true, // syncCatalog
-			options.ServerTLSConfiguration)
+			options.HTTPSTLSConfiguration)
```

Comment:
This is a bug. It is from Hao's PR. We missed it because of a lack of testing. 

I manually change it to make this branch work. But the right way is through a MR in gitlab.